src/prefect/server/database/migrations/versions/sqlite/2023_03_01_165551_f3df94dca3cc_remove_flowrun_deployment_fk.py

```diff
@@ -0,0 +1,35 @@
+"""remove_flowrun_deployment_fk
+
+Revision ID: f3df94dca3cc
+Revises: 8d148e44e669
+Create Date: 2023-03-01 16:55:51.274067
+
+"""
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "f3df94dca3cc"
+down_revision = "8d148e44e669"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.execute("PRAGMA foreign_keys=OFF")
```

Comment:
do these need to be turned back on after?  I usually do but maybe it's not necessary

Reply:
I think it might be necessary or they'll remain off for that database connection?

Reply:
woops good call!